src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: {
+    books: [],
+  },
+  reducers: {
+    addBook: (state, action) => ({
+      ...state,
+      books: [
+        {
+          id: action.payload.id,
+          title: action.payload.title,
```

Comment:
- [ ] Great job working on this milestone. For consistency reason and since this milestone is a continuation of a big project, you should have the author parameter here, as it is one of the parameter as per your book component.
 ` author: action.payload.author,`